rules/S7074/javascript/metadata.json

```
@@ -0,0 +1,23 @@
+{
+  "title": "webSecurity should be enabled",
+  "type": "VULNERABILITY",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+  ],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-7074",
+  "sqKey": "S7074",
+  "scope": "All",
+  "defaultQualityProfiles": ["Sonar way"],
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "SECURITY": "MEDIUM"
+    },
+    "attribute": "CONVENTIONAL"
```

```suggestion
    "attribute": "COMPLETE"
```
I think this is the right attribute, not 100% sure though.

---

I think that CONVENTIONAL is better in this case. The rule is not about an edge case and similar. It is a deliberate choice of the developer.